APIWorking.js

```diff
@@ -1,65 +1,80 @@
-                if (!response.ok) {
-                  console.log(response.status);
-                }
-                return response.json();
-              })
-              .then(function (res) {
-                const resultLength = res.totalResults;
-                document.getElementById("main").innerText = `Showing 10 of ${resultLength} results:` 
-                //TODO Could use a forEach instead of a for loop? Look into later
-                for (let i = 0; i < res.Search.length; i++) {
-                  //creat variables for each index of results
-                  let resultsTitle = res.Search[i].Title
-                  let resultsType = res.Search[i].Type
-                  let resultsYear = res.Search[i].Year
-                  // code to create divs to hold search results
-                  const main = document.getElementById("main");
-                  const div = document.createElement("div")
-                  //Add search results to the text of the new div
-                  let textContent = document.createTextNode(`${resultsTitle} (${resultsType}, ${resultsYear})`);
-                  const newDiv = document.getElementById("main").appendChild(div);
-                  let poster = res.Search[i].Poster;
-                  let img = document.createElement("img");
-                  let linebr = document.createElement("br")
-                  img.src = poster;
-                  img.title = "movie poster (broken image link)";
-                  newDiv.appendChild(textContent)
-                  newDiv.appendChild(linebr)
-                  newDiv.appendChild(img)
-                  document.getElementById("main").appendChild(newDiv);
-                  newDiv.classList.add("results")
-                  
-                  // Console log test to make sure it's working:
-                  // console.log(`Title: ${resultsTitle} Type: ${resultsType} Year: ${resultsYear}`)
-                }
-              });
-            } else {
-                document.getElementById("main").innerText = "Please enter 3 or more characters!"
-            }};
-        }   
-          });
\ No newline at end of file
+  .getElementById("test")
+  .addEventListener("click", function fetchTest() {
+    //remove prior results if any
+    const node = document.getElementById("main");
+    node.querySelectorAll('*').forEach(n => n.remove());
+    //set up variables for search results, with url snippets
+    const title = "&s=" + document.getElementById("searchBox").value;
+    const radios = document.getElementsByName("type");
+    const year = `&y=${document.getElementById("yearDropdown").value}`
+    //make type equal to the checked radio button.
+    for (var radio of radios) {
+      if (!radio.checked) {
+        continue;
+      }
+
+      if (title.length < 5) {
+        document.getElementById("main").innerText = "Please enter 5 or more characters!"
+        return
+      }
+
+      const type = "&type=" + radio.value;
```

Comment:
It probably would have been better to use a template literal here for consistency with the rest of the places we're concatenating strings
```suggestion
      const type = `&type=${radio.value}`;
```